Update documentation on MBM pre-trained models (#479)
ODINN-SciML/ODINN.jl: docs/src/models.md

mlp = load_model("norway_nongeo") # fast retrieval by name
```

> We also provide an API to download pre-trained models which are stored on our [HuggingFace MLP repository](https://huggingface.co/MassBalanceMachine/MLP).

Reviewer: ```suggestion
We also provide an API to download pre-trained models which are stored in our [HuggingFace MLP repository](https://huggingface.co/MassBalanceMachine/MLP).
```

> We also provide an API to download pre-trained models which are stored on our [HuggingFace MLP repository](https://huggingface.co/MassBalanceMachine/MLP).

Reviewer: ```suggestion
Internally, `MassBalanceMachine.jl` converts those exported Pytorch MLPs into `Lux.jl`, to be compatible with the whole Julia and `ODINN.jl` ecosystem. 

We also provide an API to download pre-trained models which are stored on our [HuggingFace MLP repository](https://huggingface.co/MassBalanceMachine/MLP).
```

> This API simply retrieve the `params.json` and `model.json` files associated to one pre-trained model.

Reviewer: ```suggestion
This API simply retrieves the `params.json` and `model.json` files associated to a pre-trained model. 
```
